openml-lightgbm/lightgbm-provider/src/main/java/com/feedzai/openml/provider/lightgbm/SWIGTrainData.java

```diff
@@ -118,6 +158,14 @@ public void addLabelValue(float value) {
         this.swigLabelsChunkedArray.add(value);
     }
 
+    /**
+     * Adds a value to the constraint group ChunkedArray.
+     * @param value the value to add.
+     */
+    public void addConstraintGroupValue(float value) {
+        this.swigConstraintGroupChunkedArray.add(value);
```

Comment:
if `fairnessConstrained = false` in the constructor this method will throw a NPE